com.unity.render-pipelines.high-definition/Documentation~/Planar-Reflection-Probe.md

```diff
@@ -84,3 +86,8 @@ You can use Scene view gizmos to visually customize specific properties.
 | ![](Images/ReflectionProbeGizmo4.png) | **Mirror Position**                   | Changes the behavior of the Move Tool so that it alters the **Mirror** **Position** property, rather than the **Position** of the **Transform**. |
 | ![](Images/ReflectionProbeGizmo5.png) | **Mirror Rotation**                   | Changes the behavior of the Rotate Tool so that it alters the **Mirror Rotation** property, rather than the **Rotation** of the **Transform**. |
 | ![](Images/ReflectionProbeGizmo6.png) | **Chrome Gizmo**.                     | Displays a chrome quad to preview the probe's texture in the scene. |
+
+## Best practices
+
+When the planar reflection probe is used as a mirror (i.e its influence volume overlap an object with a Material which have a smoothness and metallic parameters of 1) it is recommended to disable the rough refractions option in order to save GPU time.
+If a receiving surface isn't perfectly mirror and the rough reflections is disabled, the surface will still render as smooth but the result will be physically incorrect.
```

Comment:
If a receiving surface isn't a perfect mirror and the **Rough Reflection** option is disabled, the surface still renders smooth, but the result is physically incorrect.